src/components/Timeline.vue

```diff
@@ -497,33 +511,104 @@
     onMouseMove(event);
   }
 
-  function getPositionInMsOfMouseEvent (event: MouseEvent | PointerEvent) {
-    const mousePosXPercentage = (event.clientX - containerLeft.value) / containerWidth.value;
+  /**
+   * Returns the position in the timeline where the mouse, pointer or touch event occurred.
+   * When multiple touch points are involved, the average position is returned.
+   */
+  function getPositionInMsOfUIEvent (event: MouseEvent | PointerEvent | TouchEvent) {
+    let xPos: number;
+    if (event instanceof TouchEvent) {
+      const averageX = Array.from(event.touches).reduce((sum, touch) => sum + touch.clientX, 0) / event.touches.length;
+      xPos = averageX;
+    }
+    else {
+      xPos = event.clientX;
+    }
+    const mousePosXPercentage = (xPos - containerLeft.value) / containerWidth.value;
     return viewportStart.value + viewportDuration.value * mousePosXPercentage;
   }
 
-  function onPointerMove (event: PointerEvent, item: GTimelineItem | GTimelineMarker | null = null) {
-    emit('pointermove', { time: getPositionInMsOfMouseEvent(event), event, item });
+  const { state: touchState, setLastTouchX, setInitialTouchList, animateViewport } = useTouchEvents({ viewportStart, viewportEnd });
```

Comment:
```suggestion
  const { state: touchState, setLastTouchX, setInitialTouchList } = useTouchEvents({ viewportStart, viewportEnd });
```

**animateViewport** is assigned here but never used, which leads to a build error.

Reply:
Thank you! 👍